boards/x86/qemu_x86/doc/board.rst

```diff
@@ -63,3 +62,35 @@ The following platform features are unsupported:
 * Inter-Integrated Circuit (I2C)
 * Ethernet
 * Supervisor Mode Execution Protection (SMEP)
+
+Programming and Debugging
+*************************
+
+Use this board configuration to run basic Zephyr applications and kernel tests
+in an emulated enviroments. For example:
+
+.. code-block:: console
+
+        $ make -C samples/synchronization run
```

Comment:
the other docs specify a `BOARD=qemu_xxx` option.  Shouldn't this one do the same?